loopout.go

```diff
@@ -1020,26 +1020,43 @@ func (s *loopOutSwap) waitForConfirmedHtlc(globalCtx context.Context) (
 	return txConf, nil
 }
 
-// waitForHtlcSpendConfirmed waits for the htlc to be spent either by our own
+// waitForHtlcSpendConfirmedV2 waits for the htlc to be spent either by our own
 // sweep or a server revocation tx. During this process, this function will try
 // to spend the htlc every block by calling spendFunc.
 //
 // TODO: Improve retry/fee increase mechanism. Once in the mempool, server can
 // sweep offchain. So we must make sure we sweep successfully before on-chain
 // timeout.
-func (s *loopOutSwap) waitForHtlcSpendConfirmed(globalCtx context.Context,
+func (s *loopOutSwap) waitForHtlcSpendConfirmedV2(globalCtx context.Context,
```

Comment:
Is there a need to rename?

Reply:
since a big piece of codebase was lifted and moved to sweepbatcher maybe it would be nice to reflect this overhaul on the func name?